src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name();
```

Comment:
I suggest using `toLowerCase()` here, so that you'll get a proper color output

for example:
```
Color.WHITE.name() -> "WHITE"
Color.WHITE.name().toLowerCase() -> "white"
```

```suggestion
        return Color.values()[index].name().toLowerCase();
```